marchgazette.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
can you delete this file?

Reply:
what file

Reply:
this one, the one I'm commenting on... this file is a duplicate of the one you are trying to create, right?
if so, go to <https://github.com/Raidencio/osu-wiki/blob/march-gazette/marchgazette.md> and click on the trash icon

Reply:
done :3